tests/command_line/test_dlsnxs2cbf.py

```diff
@@ -38,6 +42,22 @@ def test_dlsnxs2cbf(dials_data, tmp_path):
             )
 
 
+@pytest.mark.parametrize("remove_axis", ["phi", "chi"], ids=["no phi", "no chi"])
+def test_dlsnxs2cbf_deleted_axis(dials_data, tmp_path, remove_axis):
+    """Check that a master file without φ or χ axes processes OK."""
+    screen = dials_data("thaumatin_eiger_screen", pathlib=True)
+    master = "Therm_6_1_master.h5"
+    links = (path.name for path in screen.glob("*") if path.name != master)
+    for name in links:
+        (tmp_path / name).symlink_to(screen / name)
+    shutil.copy(screen / master, tmp_path / master)
+
+    with h5py.File(tmp_path / master, "r+") as f:
+        del f[f"entry/sample/transformations/{remove_axis}"]
```

Comment:
Note that this is not a clean removal of each axis from the NeXus tree — I haven't gone looking for axes that `depend_on` the removed axis and tidied up the dependency tree accordingly.  I don't know if that's something we think would be sensible to do, for completeness.  `dlsnxs2cbf` doesn't care, so it would only be correctness for correctness' sake.

Reply:
I think this is fine. You can't test everything at once.